api_app/db/repositories/airlock_requests.py

```diff
@@ -103,12 +103,13 @@ def create_airlock_request_item(self, airlock_request_input: AirlockRequestInCre
 
         return airlock_request
 
-    def get_airlock_requests(self, workspace_id: str, user_id: str = None, type: AirlockRequestType = None, status: AirlockRequestStatus = None, order_by: str = None, order_ascending=True) -> List[AirlockRequest]:
-        query = self.airlock_requests_query() + f' where c.workspaceId = "{workspace_id}"'
+    def get_airlock_requests(self, workspace_id: str, initiator_user_id: str = None, type: AirlockRequestType = None, status: AirlockRequestStatus = None, order_by: str = None, order_ascending=True) -> List[AirlockRequest]:
+        query = self.airlock_requests_query() + f' WHERE c.workspaceId = "{workspace_id}"'
 
         # optional filters
-        if user_id:
-            query += ' AND c.user.id=@user_id'
+        if initiator_user_id:
+            # If the resource has history, get the first user object (initiator)
+            query += ' AND (ARRAY_LENGTH(c.history) > 0? c.history[0].user.id=@user_id: c.user.id=@user_id)'
```

Comment:
Do we need to do this to get the user who initiated the request? Or is that the value that is stored in the top-level `user` property?

Reply:
So I've opted not to change the behaviour of adding the latest user to the root user object as this is what all other resources do in TRE and is how the history is populated, so instead I've adapted the Cosmos query to query for the first user in the history if it exists, or the root level if not

Reply:
Yeah I’m with you - currently its coupled with the state of the resource at that time, so anything in the root of a resource is the current state, and history records the state as it was alongside the user that modified it, so would be a bigger undertaking to change this. It may well be we should think about renaming some fields to make this more obvious in the very least. As per our Teams discussion shall resolve this and raise a discussion on our next stand-up